public/static/docs/command-reference/metrics/diff.md

```diff
@@ -0,0 +1,123 @@
+# metrics diff
+
+Show a table of changes in [metrics](/doc/command-reference/metrics#description)
+between two versions of the <abbr>DVC repository</abbr>.
+
+> This command requires that the <abbr>project</abbr> is a
+> [Git](https://git-scm.com/) repository.
+
+## Synopsis
+
+```usage
+usage: dvc metrics diff [-h] [-q | -v]
+                        [--targets [TARGETS [TARGETS ...]]]
+                        [-t TYPE] [-x XPATH] [-R] [--show-json]
+                        [a_ref] [b_ref]
+
+positional arguments:
+  a_ref     Git reference to the older version to compare
+            (defaults to HEAD)
+  b_ref     Git reference to the newer version to compare
+            (defaults to the current workspace including changes)
+```
+
+## Description
+
+The changes shown by this command includes the new value, and numeric difference
+(delta) from the previous value of metrics (with 3-digit accuracy). They're
+calculated between two commit SHA hashes, branch or tag names, etc.
```

Comment:
`between two commits (revisions[link] - branch name, commit hash, tag etc)`

Reply:
OK 👍 Will update as part of #933 (review) above.